src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -1011,7 +1096,9 @@ public void saveResolutions(int series)
       LOGGER.info("Using {} pyramid resolutions", resolutions);
       sizeX = workingReader.getSizeX();
       sizeY = workingReader.getSizeY();
+      sizeZ = workingReader.getSizeZ();
       imageCount = workingReader.getImageCount();
+      imageCount /= sizeZ;
```

Comment:
Image count is being used to calculate the total number of tiles on line https://github.com/glencoesoftware/bioformats2raw/pull/81/files#diff-156407914fc2609796db393053797aa5ce0446652d302b8bb7a6885b6adfa642R1176

As that calculation now takes into account chunked Z the imageCount needs reduced to not include Z. It might be clearer to create a new variable further on down with the count of C and T planes.